### DISCUSSION TITLE
Task Logs

### DISCUSSION THREAD

**Discussion by venkat-katepalli** [apache/airflow#32507] [Q&A]
Hi,

I need help with capturing Task logs and send an email notification. 
My Use-case:- I am running a Job using GlueJobOperator on Failure, i like to get an email with exact error Message.

Is there a way to implement in DAG code?

Regards,
Venkat

**Comment by andresitorusz** [comment]
You can use the `on_failure_callback` parameter in your `GlueJobOperator` task to call the task failure email alert function, then use `context('exception')` in the failure email alert function to get the error/exception message thrown by Airflow.

This blog might help -> https://blog.knoldus.com/apache-airflow-automate-email-alerts-for-task-status/
